ckanext/geocat/utils/ogdch_map_utils.py

```diff
@@ -16,14 +16,10 @@ def map_geocat_to_ogdch_identifier(geocat_identifier, organization_slug):
     return '@'.join([geocat_identifier, organization_slug])
 
 
-def map_to_ogdch_publisher(geocat_publisher, organization_slug):
+def map_to_ogdch_publisher(geocat_publisher):
```

Comment:
This function can be simplieifed, now that the geocat publisher is mapped with a url